examples/resources/grafana_data_source_config_lbac_rules/resource.tf

```diff
@@ -0,0 +1,32 @@
+resource "grafana_team" "team" {
+  name = "Team Name"
+}
+
+resource "grafana_data_source" "test" {
+  type                = "loki"
+  name                = "loki-from-terraform"
+  url                 = "https://mylokiurl.net"
+  basic_auth_enabled  = true
+  basic_auth_username = "username"
+
+  json_data_encoded = jsonencode({
+    authType = "default"
+    ## basicAuthPassword = "<>"
```

Comment:
Is this line needed?

Reply:
it's not needed at all. just showing that you need to have a password to connect with the datasource from basicAuth